metis-sys/build.rs

```diff
@@ -129,9 +129,6 @@ fn build_lib() {
     #[cfg(any(not(debug_assertions), feature = "force-optimize-vendor"))]
     build.define("NDEBUG", None).define("NDEBUG2", None);
 
-    #[cfg(feature = "force-optimize-vendor")]
-    build.no_default_flags(true).opt_level(3).debug(false);
-
```

Comment:
I did not look in-depth into the code, but throwing a refactor on the name of the feature seems to indicate that these were its only use. Maybe the feature can be removed if vendor optimizations aren't introduced back to the build script.

Reply:
I do not fully understand. The feature `force-optimize-vendor` is used just two lines before, so it still exists.

Reply:
I'm not sure about how the crate is used, but the above condition is always met as long as its compiled in release mode. `debug_assertions` is a builtin configuration option that, by default, is only enabled if no optimizations are enabled. Unless this behavior does not carry over in custom built script (which I am clueless about), compiling in release mode already ensures this line is included.

ref on `debug_assertions`: https://doc.rust-lang.org/reference/conditional-compilation.html#debug_assertions

Reply:
Ok I see the issue, you can forget about all my comments :)